utils.py

```diff
@@ -48,6 +49,18 @@ def waitForUserActivity():
     while firstInputInfo - win32api.GetLastInputInfo() == 0:
         pass
 
+def countdown(t):
+    while t:
+        mins, secs = divmod(t, 60)
+        timeformat = '{:02d}:{:02d}'.format(mins, secs)
+        print(f"Next break in {timeformat}", end="\r")
```

Comment:
If you added a sendNotification here, maybe you could have an updating notification every second, did you try that out?

Reply:
Yes. It's not possible - overwriting messages is not supported by sendNotification